src/components/editor-page/splitter/splitter.tsx

```diff
@@ -48,42 +44,43 @@ const extractHorizontalPosition = (moveEvent: MouseEvent | TouchEvent): number |
   }
 }
 
+const snapPercentage = 10
```

Comment:
Maybe this should be
```suggestion
const SNAP_PERCENTAGE = 10
```